### DISCUSSION TITLE
iOS App - Cannot assign issues

### DISCUSSION THREAD

**Discussion by navidj1** [community/community#18034] [Mobile] [answered]
I setup a repository and added my developer under settings>collaborators, and he accepted the invitation.

I created an issue in the **iOS app**, but when I try to assign it to my collaborator, I **only** see myself on the list.

It's evident that this is a problem with the **app** because I can see and assign issues to my colleague without any problems on the **website**.

**Comment by Jino0910** [comment]
General -> Access -> Collaborators 

Did you confirm **DIRECT ACCESS** ?

<img width="1672" alt="스크린샷 2022-06-16 오후 6 06 28" src="https://user-images.githubusercontent.com/25897138/174035288-d466b62f-9a9d-4249-98e9-e939d7d03d2d.png">

**Reply by navidj1** [reply]
Yes, 
I assume the access is setup correctly as on the web I have no problem,  it’s just iPhone app that only shows myself not others .
Currently I post issues on the app then I logging to website and assign it.thanks

**Comment by JoshuaLuckers** [comment, marked as answer]
I believe the dropdown shows suggestions, did you try to search for the user in the search bar?

**Reply by navidj1** [reply]
Oh thank you! I never searched. It was coming up on the web without search and I assumed it’s the same . Thank you Joshua

**Reply by JoshuaLuckers** [reply]
You're welcome, from time to time it's confusing me as well.
